pages/index.js

```diff
@@ -1,9 +1,21 @@
 import Footer from "~/components/Footer";
+import Header from "~/components/Header";
+import headerImg from "../public/images/headerImg.jpeg";
+import Image from "next/image"
+
+const pageHeader = {
+  backgroundColour: "yellow",
+  imgSrc: headerImg,
+  imgAlt: "girl on a beach",
+  title: "Guiding you through mental health resources.",
+  caption:
+    "Miricyl helps you find key, informative resources to guide you on your mental health journey",
+};
 
 const Index = () => {
   return (
     <div>
-      <h2>Welcome to NextJS!</h2>
+      <Header pageHeader={pageHeader} />
```

Comment:
I see you've defined this prop object above. I wonder if it would be nicer if we could just pass the props in individually?

```
<Header backgroundColor="yellow" imgSrc={headerImg} title="Guiding you through mental health resources" />
```

What do you think?
```

Reply:
I thought it's the best practice to do it this way. Great to know. Thank you

Reply:
It depends. If it's a dataset, it can be best to pass it as an object. But these props all relate to different parts of the component they're going into, and it makes it clearer what the component can take.

Reply:
Oh I see, makes sense, thank you.